### ISSUE TITLE
[Bug] MyPet exception error on /trade accept

### ISSUE THREAD

**Issue by mrfloris** [Zrips/TradeMeAddon#40]
Installed fresh paper 1.16.4, fresh download from spigotmc of trademe, i got cmi's latest version from spigotmc, and zrips compiled vault. Run this as a new server, new world. did /stop when i saw everything loaded fine. and started it up again.

I got mypet 3.11 from dev build here: https://mypet-plugin.de/download, it says R3 is supported, 3.10 is for 1.15.2 (and probably what trademe supports)

Initially starting it back up and doing trades it works fine. It goes wrong once someone has purchased something as operator from /petshop

Reported on discord, confirmed by me just now. 
https://ptb.discord.com/channels/452792793631555594/713838991744434277/775678220259164160

<img width="1063" alt="Screenshot 2020-11-10 at 12 29 46" src="https://user-images.githubusercontent.com/28841349/98668616-6c85dc80-2350-11eb-90f9-f56954c1a57e.png">
<img width="1672" alt="Screenshot 2020-11-10 at 12 30 03" src="https://user-images.githubusercontent.com/28841349/98668639-76a7db00-2350-11eb-988c-33440466b6b1.png">

**Comment by mrfloris** [comment]
```
floris@Floriss-iMac trademetest % ./1MB-minecraft.sh
System Info: Java 11 (OpenJDK 64-Bit Server VM 11.0.2+9) Host: Mac OS X 10.15.7 (x86_64)
Loading libraries, please wait...
[12:22:19 INFO]: [STDERR]: You have used the Spigot command line EULA agreement flag.
[12:22:19 INFO]: [STDERR]: By using this setting you are indicating your agreement to Mojang's EULA (https://account.mojang.com/documents/minecraft_eula).
[12:22:19 INFO]: [STDERR]: If you do not agree to the above EULA please stop your server and remove this flag immediately.
[12:22:19 INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[12:22:20 INFO]: Reloading ResourceManager: Default, bukkit
[12:22:20 INFO]: Loaded 7 recipes
[12:22:21 INFO]: Starting minecraft server version 1.16.4
[12:22:21 INFO]: Loading properties
[12:22:21 INFO]: This server is running Paper version git-Paper-267 (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)
[12:22:21 INFO]: Server Ping Player Sample Count: 12
[12:22:21 INFO]: Using 4 threads for Netty based IO
[12:22:21 INFO]: Debug logging is disabled
[12:22:21 INFO]: Default game type: SURVIVAL
[12:22:21 INFO]: Generating keypair
[12:22:21 INFO]: Starting Minecraft server on *:55555
[12:22:21 INFO]: Using default channel type
[12:22:22 INFO]: [Vault] Loading Vault v1.7.3-b${env.TRAVIS_BUILD_NUMBER}
[12:22:22 INFO]: [CMI] Loading CMI v8.7.10.4
[12:22:22 INFO]: [MyPet] Loading MyPet v3.11
[12:22:22 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Error-Reporter ENABLED
[12:22:22 INFO]: [TradeMe] Loading TradeMe v6.0.5.1
[12:22:22 INFO]: Server permissions file permissions.yml is empty, ignoring it
[12:22:22 INFO]: [Vault] Enabling Vault v1.7.3-b${env.TRAVIS_BUILD_NUMBER}
[12:22:22 WARN]: [Vault] Loaded class com.Zrips.CMI.Modules.Economy.Economy from CMI v8.7.10.4 which is not a depend, softdepend or loadbefore of this plugin.
[12:22:22 INFO]: [Vault] [Economy] CMI Economy found: Waiting
[12:22:22 INFO]: [Vault] [Permission] SuperPermissions loaded as backup permission system.
[12:22:22 INFO]: [Vault] Enabled Version 1.7.3-b${env.TRAVIS_BUILD_NUMBER}
[12:22:22 INFO]: Preparing level "spawn"
[12:22:22 INFO]: -------- World Settings For [spawn] --------
[12:22:22 INFO]: Item Merge Radius: 2.5
[12:22:22 INFO]: Item Despawn Rate: 6000
[12:22:22 INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[12:22:22 INFO]: Nerfing mobs spawned from spawners: false
[12:22:22 INFO]: Mob Spawn Range: 8
[12:22:22 INFO]: Experience Merge Radius: 3.0
[12:22:22 INFO]: Cactus Growth Modifier: 100%
[12:22:22 INFO]: Cane Growth Modifier: 100%
[12:22:22 INFO]: Melon Growth Modifier: 100%
[12:22:22 INFO]: Mushroom Growth Modifier: 100%
[12:22:22 INFO]: Pumpkin Growth Modifier: 100%
[12:22:22 INFO]: Sapling Growth Modifier: 100%
[12:22:22 INFO]: Beetroot Growth Modifier: 100%
[12:22:22 INFO]: Carrot Growth Modifier: 100%
[12:22:22 INFO]: Potato Growth Modifier: 100%
[12:22:22 INFO]: Wheat Growth Modifier: 100%
[12:22:22 INFO]: NetherWart Growth Modifier: 100%
[12:22:22 INFO]: Vine Growth Modifier: 100%
[12:22:22 INFO]: Cocoa Growth Modifier: 100%
[12:22:22 INFO]: Bamboo Growth Modifier: 100%
[12:22:22 INFO]: SweetBerry Growth Modifier: 100%
[12:22:22 INFO]: Kelp Growth Modifier: 100%
[12:22:22 INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
[12:22:22 INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
[12:22:22 INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 End City: 10387313 Slime: 987234911 Bastion: 30084232 Fortress: 30084232 Mansion: 10387319 Fossil: 14357921 Portal: 34222645
[12:22:22 INFO]: Max TNT Explosions: 100
[12:22:22 INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[12:22:22 INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[12:22:22 INFO]: Zombie Aggressive Towards Villager: true
[12:22:22 INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
[12:22:22 INFO]: View Distance: 4
[12:22:23 INFO]: -------- World Settings For [spawn_the_end] --------
[12:22:23 INFO]: Item Merge Radius: 2.5
[12:22:23 INFO]: Item Despawn Rate: 6000
[12:22:23 INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[12:22:23 INFO]: Nerfing mobs spawned from spawners: false
[12:22:23 INFO]: Mob Spawn Range: 8
[12:22:23 INFO]: Experience Merge Radius: 3.0
[12:22:23 INFO]: Cactus Growth Modifier: 100%
[12:22:23 INFO]: Cane Growth Modifier: 100%
[12:22:23 INFO]: Melon Growth Modifier: 100%
[12:22:23 INFO]: Mushroom Growth Modifier: 100%
[12:22:23 INFO]: Pumpkin Growth Modifier: 100%
[12:22:23 INFO]: Sapling Growth Modifier: 100%
[12:22:23 INFO]: Beetroot Growth Modifier: 100%
[12:22:23 INFO]: Carrot Growth Modifier: 100%
[12:22:23 INFO]: Potato Growth Modifier: 100%
[12:22:23 INFO]: Wheat Growth Modifier: 100%
[12:22:23 INFO]: NetherWart Growth Modifier: 100%
[12:22:23 INFO]: Vine Growth Modifier: 100%
[12:22:23 INFO]: Cocoa Growth Modifier: 100%
[12:22:23 INFO]: Bamboo Growth Modifier: 100%
[12:22:23 INFO]: SweetBerry Growth Modifier: 100%
[12:22:23 INFO]: Kelp Growth Modifier: 100%
[12:22:23 INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
[12:22:23 INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
[12:22:23 INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 End City: 10387313 Slime: 987234911 Bastion: 30084232 Fortress: 30084232 Mansion: 10387319 Fossil: 14357921 Portal: 34222645
[12:22:23 INFO]: Max TNT Explosions: 100
[12:22:23 INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[12:22:23 INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[12:22:23 INFO]: Zombie Aggressive Towards Villager: true
[12:22:23 INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
[12:22:23 INFO]: View Distance: 4
[12:22:23 INFO]: Preparing start region for dimension minecraft:overworld
[12:22:23 INFO]: Loaded 0 spawn chunks for world spawn
[12:22:23 INFO]: Preparing spawn area: 0%
[12:22:23 INFO]: Time elapsed: 530 ms
[12:22:23 INFO]: Preparing start region for dimension minecraft:the_end
[12:22:23 INFO]: Loaded 0 spawn chunks for world spawn_the_end
[12:22:23 INFO]: Preparing spawn area: 0%
[12:22:23 INFO]: Time elapsed: 315 ms
[12:22:23 INFO]: [CMI] Enabling CMI v8.7.10.4
[12:22:24 INFO]: [CMIPaperLib] Enabling CMIPaperLib v1.0.0
[12:22:25 INFO]: ┏━━━┓ ┏━┓┏━┓ ┏━━┓
[12:22:25 INFO]: ┃┏━┓┃ ┃ ┗┛ ┃ ┗┫┣┛
[12:22:25 INFO]: ┃┃ ┗┛ ┃┏┓┏┓┃  ┃┃
[12:22:25 INFO]: ┃┃ ┏┓ ┃┃┃┃┃┃  ┃┃
[12:22:25 INFO]: ┃┗━┛┃ ┃┃┃┃┃┃ ┏┫┣┓
[12:22:25 INFO]: ┗━━━┛ ┗┛┗┛┗┛ ┗━━┛
[12:22:25 INFO]: _______________________________________________________
[12:22:25 INFO]:   Integrating PaperSpigot async methods
[12:22:26 INFO]:   51 Enabled and 0 Disabled modules
[12:22:26 INFO]:   Permission plugin: Vault
[12:22:26 INFO]:   Initialized Cipher256 AES
[12:22:26 INFO]:   Loaded (52) regular alias into memory. Took 30ms
[12:22:26 INFO]:   Loaded (1) custom text's into memory. Took 3ms
[12:22:27 INFO]:   3.32.3 data base type detected
[12:22:27 INFO]:   Started SqLite data base. Took 276ms
[12:22:27 INFO]:   Vault was found - Enabling capabilities. Economy: CMIEconomy
[12:22:27 INFO]:   Loaded (5) warning categories into memory. Took 1ms
[12:22:27 INFO]:   Loaded (3) warning commands into memory. Took 0ms
[12:22:27 INFO]:   Loaded (131) custom mob heads into memory. Took 13ms
[12:22:27 INFO]:   Loaded (0) cooldowns into memory. Took 18ms
[12:22:27 INFO]:   Initializing BungeeCord
[12:22:27 WARN]: Initializing Legacy Material Support. Unless you have legacy plugins and/or data this is a bug!
[12:22:30 INFO]:   Loaded (5) kits into memory. Took 3429ms
[12:22:30 INFO]:   Loaded (7) ranks into memory. Took 9ms
[12:22:30 INFO]:   Loaded (8) playtime rewards into memory. Took 2ms
[12:22:30 INFO]:   Loaded (0) player data into memory. Took 5ms
[12:22:30 INFO]:   Loaded (0) playtime records into memory. Took 0ms
[12:22:30 INFO]:   Loaded (0) playtime reward records into memory. Took 0ms
[12:22:31 INFO]:   Registered events. Took 128ms
[12:22:31 INFO]:   Loaded (0) event action commands into memory. Took 3ms
[12:22:31 INFO]:   Updated (EN) language file. Took 235ms
[12:22:31 INFO]:   Can't load worth value for LEGACY_RAW_FISH
[12:22:31 INFO]:   Can't load worth value for SIGN
[12:22:31 INFO]:   Loaded (243) worth values into memory. Took 14ms
[12:22:31 INFO]:   Vault permissions was found - Enabling capabilities.
[12:22:31 INFO]:   Starting world timer.
[12:22:31 INFO]:   Initializing world manager.
[12:22:31 INFO]:   Loaded (5) schedules into memory. Took 11ms
[12:22:31 INFO]:   Loaded GeoIP
[12:22:31 INFO]:   Loaded (0) skin cache entries into memory. Took 0ms
[12:22:31 INFO]:   Version 8.7.10.4 has been enabled
[12:22:31 INFO]: ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
[12:22:31 INFO]: [Vault][Economy] CMI Economy hooked.
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Enabling MyPet v3.11
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] ##################################################
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] #  A new build is available: 3.11-SNAPSHOT #1586 #
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] #  https://mypet-plugin.de/download              #
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] ##################################################
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Compat mode for v1_16_R3 loaded.
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Exp calculation mode: MyPet
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Loading WorldGroups...
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet]    added 'spawn' to 'default'
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet]    added 'spawn_the_end' to 'default'
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Connect to SQLite database...
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] SQLite connection successful.
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Vault (1.7.3-b${env.TRAVIS_BUILD_NUMBER}) (Economy: CMIEconomy) (Permissions: SuperPerms) hook activated.
[12:22:31 INFO]: [de.Keyle.MyPet.MyPetPlugin] [MyPet] Version 3.11-SNAPSHOT-b1585 ENABLED
[12:22:31 INFO]: [TradeMe] Enabling TradeMe v6.0.5.1
[12:22:31 INFO]: [TradeMe] Version 6.0.5.1 has been enabled
[12:22:31 INFO]: [TradeMe] MyPet plugin was found!
[12:22:31 INFO]: [TradeMe] Vault plugin was found!
[12:22:32 INFO]: [TradeMe] Connected to sqlite
[12:22:33 INFO]: [TradeMe] 0 trade logs preloaded
[12:22:33 INFO]: Running delayed init tasks
[12:22:33 INFO]: Done (11.565s)! For help, type "help"
[12:22:33 INFO]: Timings Reset
[12:22:33 INFO]: [Vault] Checking for Updates ...
[12:22:33 INFO]: [Vault] No new version available
> pl
[12:22:40 INFO]: Plugins (5): CMI, CMIPaperLib, MyPet, TradeMe, Vault
> ver
[12:22:43 INFO]: This server is running Paper version git-Paper-267 (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)
[12:22:43 INFO]: Checking version, please wait...
[12:22:43 INFO]: Previous version: git-Paper-253 (MC: 1.16.3)
[12:22:43 INFO]: You are running the latest version
[12:23:21 INFO]: UUID of player mrfloris is 631e3896-da2a-4077-974b-d047859d76bc
[12:23:21 INFO]: mrfloris joined the game
[12:23:21 INFO]: mrfloris[/127.0.0.1:61270] logged in with entity id 59 at ([spawn]227.5, 70.0, -9.5)
> cmi time freeze
[12:23:33 INFO]: Can't find world with this name!
> cmi time freeze spawn
[12:23:36 INFO]: spawn time have has been frozen
[12:23:42 INFO]: <mrfloris> td
> cmi god mrfloris true
[12:23:48 INFO]: You have set god to True for mrfloris (Online).
[12:23:51 INFO]: mrfloris issued server command: /trade
[12:23:53 INFO]: mrfloris issued server command: /trade help
[12:24:37 INFO]: UUID of player partyroom is dd3d28dd-ec7c-4b76-83c6-1acb0f68de46
[12:24:37 INFO]: partyroom joined the game
[12:24:37 INFO]: partyroom[/127.0.0.1:61293] logged in with entity id 308 at ([spawn]226.5, 70.0, -24.5)
[12:24:40 INFO]: mrfloris issued server command: /trade partyroom
[12:24:44 INFO]: partyroom issued server command: /trade accept
[12:24:51 INFO]: partyroom issued server command: /trade accept
[12:24:55 INFO]: partyroom issued server command: /trade mrfloris
[12:24:59 INFO]: mrfloris issued server command: /trade accept
[12:25:40 INFO]: mrfloris issued server command: /trade partyroom
[12:25:44 INFO]: partyroom issued server command: /trade accept
[12:26:04 INFO]: partyroom issued server command: /trade
[12:26:14 INFO]: partyroom issued server command: /mypet
[12:26:17 INFO]: partyroom issued server command: /petshop
> op mrfloris
[12:26:22 INFO]: Made mrfloris a server operator
> op partyroom
[12:26:24 INFO]: Made partyroom a server operator
[12:26:26 INFO]: partyroom issued server command: /petshop
> WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by de.Keyle.MyPet.api.util.ReflectionUtil (file:/Users/floris/MinecraftServer/trademetest/plugins/MyPet-3.11-SNAPSHOT-B1586.jar) to field java.lang.reflect.Field.modifiers
WARNING: Please consider reporting this to the maintainers of de.Keyle.MyPet.api.util.ReflectionUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[12:26:39 INFO]: partyroom issued server command: /mypet help
[12:26:49 INFO]: partyroom issued server command: /trade mrfloris
[12:26:54 INFO]: mrfloris issued server command: /trade accept
[12:26:54 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'trade' in plugin TradeMe v6.0.5.1
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.16.4.jar:git-Paper-267]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.16.4.jar:git-Paper-267]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:802) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.PlayerConnection.handleCommand(PlayerConnection.java:2007) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.PlayerConnection.c(PlayerConnection.java:1818) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1771) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1136) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1129) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1090) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1004) ~[patched_1.16.4.jar:git-Paper-267]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.4.jar:git-Paper-267]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NoSuchMethodError: de.Keyle.MyPet.api.entity.MyPet.getSkilltree()Lde/Keyle/MyPet/api/skill/skilltree/SkillTree;
	at me.Zrips.TradeMe.SpecialActionPack.MyPetUtil.Buttons(MyPetUtil.java:164) ~[?:?]
	at me.Zrips.TradeMe.Util.updateAmountOffer(Util.java:1002) ~[?:?]
	at me.Zrips.TradeMe.commands.list.accept.perform(accept.java:81) ~[?:?]
	at me.Zrips.TradeMe.commands.CommandsHandler.onCommand(CommandsHandler.java:195) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.16.4.jar:git-Paper-267]
	... 19 more
[12:28:48 INFO]: mrfloris lost connection: Disconnected
[12:28:48 INFO]: mrfloris left the game
[12:28:53 INFO]: partyroom lost connection: Disconnected
[12:28:53 INFO]: partyroom left the game
>
> pl
[12:29:26 INFO]: Plugins (5): CMI, CMIPaperLib, MyPet, TradeMe, Vault
> ver
[12:29:27 INFO]: This server is running Paper version git-Paper-267 (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)
[12:29:27 INFO]: Previous version: git-Paper-253 (MC: 1.16.3)
[12:29:27 INFO]: You are running the latest version
> ver CMI
[12:29:30 INFO]: CMI version 8.7.10.4
[12:29:30 INFO]: Powerful server management tool
[12:29:30 INFO]: Website: https://www.spigotmc.org/resources/content-management-interface.3742/
[12:29:30 INFO]: Author: Zrips
> cmi version
[12:29:33 INFO]: --------------------------------------------------
[12:29:33 INFO]: CMI: 8.7.10.4 SqLite
[12:29:33 INFO]: Server: Paper(267) 1.16.4-R0.1-SNAPSHOT
[12:29:33 INFO]: CMI economy: True Vault: 1.7.3-b CMI Chat: False
[12:29:33 INFO]: Modules -> 50 enabled 1 disabled: votifier
[12:29:33 INFO]: --------------------------------------------------
> ver MyPet
[12:29:37 INFO]: MyPet version 3.11
[12:29:37 INFO]: Website: https://www.spigotmc.org/resources/mypet.12725/
[12:29:37 INFO]: Author: Keyle
> ver TradeMe
[12:29:39 INFO]: TradeMe version 6.0.5.1
[12:29:39 INFO]: Powerfull trade plugin
[12:29:39 INFO]: Website: https://www.spigotmc.org/resources/trademe.7544/
[12:29:39 INFO]: Author: Zrips
>
```

**Comment by Zrips** [comment]
Fixed in latest builds
